modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp

```diff
@@ -365,6 +365,8 @@ SearchContainer<?> searchContainer = new SearchContainer(renderRequest, itemSele
 									<%
 									FileVersion latestFileVersion = fileEntry.getLatestFileVersion();
 
+									boolean isEditable = repositoryEntryBrowserDisplayContext.isEditable(latestFileVersion);
```

Comment:
By convention, boolean variable names in Java drop the "is" prefix. It should be:
```boolean editable = ...```

Reply:
Thanks, updated